lib/groups/config.js

```diff
@@ -1,142 +1,136 @@
-const { existsSync } = require("fs");
-const { resolve, extname } = require("path");
+const { existsSync } = require('fs');
+const { resolve, extname } = require('path');
 
-const GroupHelper = require("../utils/group-helper");
+const GroupHelper = require('../utils/group-helper');
 
 class ConfigGroup extends GroupHelper {
-	constructor(options) {
-		super(options);
-		this.extensions = [".mjs", ".js", ".json", ".babel.js", ".ts"];
-		this.defaultConfigFiles = this.getDefaultConfigFiles();
-		this.configFiles = null;
-		this.configOptions = [];
-	}
-	getDefaultConfigFiles() {
-		const {mode} = this.args;
-		let DEFAULT_FILES = ["webpack.config", "webpack.dev", "webpackfile"];
-		if(mode === 'prod') {
-			DEFAULT_FILES.unshift("webpack.prod");
-		}
-		return DEFAULT_FILES
-			.map(filename =>
-				this.extensions.map(ext => ({
-					path: resolve(filename + ext),
-					ext: ext
-				}))
-			)
-			.reduce((a, i) => a.concat(i), []);
-	}
+    constructor(options) {
+        super(options);
+        this.extensions = ['.mjs', '.js', '.json', '.babel.js', '.ts'];
+        this.defaultConfigFiles = this.getDefaultConfigFiles();
+        this.configFiles = null;
+        this.configOptions = [];
+    }
+    getDefaultConfigFiles() {
+        const { mode } = this.args;
```

Comment:
@evenstensberg This fails because args doesn't have a proper interface.
Could you help and tell us what should we have inside `args`?

Reply:
this.args is a mimic of a partial webpack Config object and `outputOptions`

```js
this.args = {
webpackOptions: {},
outputOptions: {}
}
```

Reply:
So why there's a `mode` in there?